Exiled.API/Features/Player.cs

```diff
@@ -399,10 +399,20 @@ public RoleType Role
         /// </summary>
         public PlayerMovementState MoveState => ReferenceHub.animationController.MoveState;
 
+        /// <summary>
+        /// Gets the player's current animation.
+        /// </summary>
+        public int CurrentAnimation => ReferenceHub.animationController.curAnim;
+
+        /// <summary>
+        /// Gets a value indicating whether or not the player is sprinting.
+        /// </summary>
+        public bool IsSprinting => CurrentAnimation == 1;
```

Comment:
Please use MoveState == PlayerMovementState.Sprinting for this, not the animation number, incase the animation numbers change in the future.

Reply:
Alr